cli/index.ts

```diff
@@ -83,5 +83,23 @@ yargs
     (yargs: Argv) => {},
     (cliArgs: any) => executeCommand("keyfile", cliArgs)
   )
+  .command('logs', 'Retrieve logs on Forta Agent',
+    (yargs: Argv) => {
+      yargs
+      .option('after', {
+        description: 'An ISO timestamp representing the oldest time to include in logs',
+        type: 'string'
+      }).option('before', {
+        description: 'An ISO timestamp representing the latest time to include in logs',
+        type: 'string'
+      }).demandOption('before')
```

Comment:
would be nice to also have an optional arg to specify agentId. this would allow using this command for any agent